crates/pop-contracts/src/build.rs

```diff
@@ -1,14 +1,19 @@
 // SPDX-License-Identifier: GPL-3.0
-use contract_build::{execute, ExecuteArgs};
+use contract_build::{execute, BuildMode, ExecuteArgs};
 use std::path::PathBuf;
 
 use crate::utils::helpers::get_manifest_path;
 
-/// Build the smart contract located in the specified `path`.
-pub fn build_smart_contract(path: &Option<PathBuf>) -> anyhow::Result<String> {
+/// Build the smart contract located in the specified `path` in `build_release` mode.
+pub fn build_smart_contract(path: &Option<PathBuf>, build_release: bool) -> anyhow::Result<String> {
```

Comment:
Why not giving the `BuildMode` as argument in stead of a `bool`?

Reply:
`BuildMode` is an enum from the `contract_build` crate (cargo-contracts). The pop-cli crate calls the build_smart_contract function, which resides in the pop_contracts crate and contains all the contract-related logic. This ensures that pop-cli does not handle the contract logic directly.